src/main/resources/static/js/profile.js

```
@@ -0,0 +1,825 @@
+            isEditing = false
+            isPasswordVerified = false
+            imageAction = "keep"
+            await fetchUserInfo()
+        } else {
+            const data = await response.json()
+            const errorMessage = data.message || data.code || "정보 업데이트에 실패했습니다."
+            showToast(errorMessage, "error")
+        }
+    } catch (error) {
+        console.error("사용자 정보 업데이트 실패:", error)
+        showToast("네트워크 오류가 발생했습니다.", "error")
```

Biome (1.9.4)

[error] value is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)